restful_api/common.py

```diff
@@ -0,0 +1,65 @@
+import logging
+import datetime
+import json
+import ast
+#from urllib.parse import unquote
+
+import werkzeug.wrappers
+
+_logger = logging.getLogger(__name__)
+
+
+def default(o):
+    if isinstance(o, (datetime.date, datetime.datetime)):
+        return o.isoformat()
+
+
+def valid_response(data, status=200):
+    """Valid Response
+    This will be return when the http request was successfully processed."""
+    data = {"count": len(data), "data": data}
+    return werkzeug.wrappers.Response(
+        status=status,
+        content_type="application/json; charset=utf-8",
+        headers=[("Cache-Control", "no-store"),('Access-Control-Allow-Headers', 'Origin, Content-Type, X-Auth-Token, charset'),('Access-Control-Allow-Methods','POST, GET, OPTIONS, DELETE, PUT'), ('Access-Control-Allow-Origin', '*'), ("Pragma", "no-cache")],
```

Comment:
Do we really need to allow all HTTP methods? Don't we need just POST/GET?

Reply:
This will be the return value whenever the server runs into an error either from the client or the server.(in-case in further operation needed this HTTP types means,so commonly mentioned all the HTTP types.)